packages/react/src/file-input/FileInput.tsx

```diff
@@ -67,18 +72,26 @@ export const FileInput = forwardRef<HTMLInputElement, FileInputProps>(
 			[onChangeProp]
 		);
 
+		const fallbackHint =
+			hint ||
+			(accept &&
+				`Files accepted: ${
+					Array.isArray(accept) ? accept.join(', ') : accept
```

Comment:
Still need to map it to file extensions instead of mime types like in FileUpload. Here's how that does it:

```
// Converts an array of mime types, e.g. `image/jpeg`, `application/pdf` into a format accepted by react-dropzone
const accept = useMemo(() => {
	if (!acceptProp) return;
	return Object.fromEntries(
		acceptProp.map((item) => {
			// If the array item is a string, it's a key of `fileTypeMapping`
			if (typeof item === 'string') {
				return [item, fileTypeMapping[item].extensions];
			} else {
				return [item.mimeType, item.extensions];
			}
		})
	);
}, [acceptProp]);
```